### PR TITLE
[FLINK-15199] Fix compile error after FLINK-14926

### PR DESCRIPTION
**Pull request by pnowojski**

### CODE REVIEW

**Review by carp84 — APPROVED** [review]
+1, thanks for the fix @pnowojski !

**Comment by pnowojski** [comment]
Verified locally - merging without waiting for a slot on the travis to unblock some release blocker. Will be tested by benchmarking CI anyway.

**Pull request merged**
